uk_bin_collection/uk_bin_collection/councils/HarboroughDistrictCouncil.py

```
@@ -23,13 +23,16 @@ def parse_data(self, page: str, **kwargs) -> dict:
         check_uprn(user_uprn)
         bindata = {"bins": []}
 
-        URI = "https://harborough.fccenvironment.co.uk/detail-address"
+        URI1 = "https://harborough.fccenvironment.co.uk/"
+        URI2 = "https://harborough.fccenvironment.co.uk/detail-address"
```

Ruff (0.14.14)

[error] Local variable `URI2` is assigned to but never used

Remove assignment to unused variable `URI2`

(F841)